### DISCUSSION TITLE
Support Views commits in chronological order in Vscode in commits graph

### DISCUSSION THREAD

**Discussion by 16jjli3** [gitkraken/vscode-gitlens#2632] [Ideas]
## Support
In Gitlens, I didn't see anything about show commit in chronological order.
I think it's useful for reading projects code. 
![2023-04-14 01-29-26 的屏幕截图](https://user-images.githubusercontent.com/41035441/231838622-0f913da9-2963-417e-89fa-00cb2024ed4c.png)

**Comment by jkelroy** [comment]
Hi @16jjli3, this is a duplicate of #2562. This is not possible but you can consider submitting a feature request to support this [here](https://github.com/gitkraken/vscode-gitlens/issues/new/choose).
